service-api/app/test/AppTest/Handler/LpasActionsHandlerTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace AppTest\Handler;
+
+use App\Exception\ApiException;
+use App\Handler\LpasActionsHandler;
+use App\Service\Features\FeatureEnabled;
+use App\Service\Lpa\OlderLpaService;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Psr\Http\Message\ServerRequestInterface;
+
+class LpasActionsHandlerTest extends TestCase
+{
+    public function test_when_sirius_is_down_remove_stored_LPA_request()
```

Comment:
As a preference and to keep things consistent we use a @test docblock rather than a prefix.
```suggestion
    /**
     * @test
     */
    public function when_sirius_is_down_remove_stored_LPA_request()
```

Reply:
Or, the single line variant to keep it tidy
```suggestion
    /** @test */
    public function test_when_sirius_is_down_remove_stored_LPA_request()
```